pgjdbc/src/main/java/org/postgresql/jdbc/TimestampUtils.java

```diff
@@ -63,6 +63,8 @@
   private static final LocalDate MIN_LOCAL_DATE = LocalDate.of(4713, 1, 1).with(ChronoField.ERA, IsoEra.BCE.getValue());
   private static final LocalDateTime MIN_LOCAL_DATETIME = MIN_LOCAL_DATE.atStartOfDay();
   private static final OffsetDateTime MIN_OFFSET_DATETIME = MIN_LOCAL_DATETIME.atOffset(ZoneOffset.UTC);
+  private static final Duration PG_EPOCH_DIFF = Duration.between(Instant.EPOCH, LocalDate.of(2000, 1, 1)
+      .atStartOfDay().toInstant(ZoneOffset.UTC));
```

Comment:
```suggestion
  private static final Duration PG_EPOCH_DIFF =
      Duration.between(Instant.EPOCH, LocalDate.of(2000, 1, 1).atStartOfDay().toInstant(ZoneOffset.UTC));
```

No idea which one is better, however, having `.atStartOfDay` separated from `LocalDate.of` looks strange